repo/packages/S/scale/0/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "packagingVersion": "3.0",
+  "minDcosReleaseVersion": "1.7",
+  "scm": "https://github.com/ngageoint/scale.git",
+  "postInstallNotes": "The Scale DCOS Service has been successfully installed!\n\n\tDocumentation: https://ngageoint.github.io/scale/\n\tIssues: https://github.com/ngageoint/scale/issues",
+  "maintainer": "http://www.nga.mil",
+  "name": "scale",
+  "tags": [
+    "bigdata",
+    "batch",
+    "analytics",
+    "remotesensing",
+    "algorithms",
+    "geospatial"
+  ],
+  "framework": true,
+  "version": "4.0.0",
```

Comment:
version should be "scaleVersion-universePackageVersion". Ex: if Scale's version is 4.0.0 then version should be "4.0.0-0.0.1